javascript/node/selenium-webdriver/test/bidi/bidi_test.js

```diff
@@ -301,6 +301,7 @@ suite(
       let startIndex = 0
       let endIndex = 5
       let pdfMagicNumber = 'JVBER'
+      let pngMagicNumber = 'iVBOR'
```

Comment:
I am not sure if this is the best way to test. I am trying to figure out how to test this in Java meanwhile. Since the W3C BiDi spec doesn't mention that png is the default format of the image that will be returned. There is a TODO item regarding adding different formats.